hooks/useERC721Transfer.ts

```diff
@@ -0,0 +1,26 @@
+import { useCallback } from "react"
+import { transferFromERC721 } from "../lib/cre8or"
+import { useEthersSigner } from "./useEthersSigner"
+
+const useERC721Transfer = ({ afterTransfer }) => {
+  const signer = useEthersSigner()
+
+  const transferERC721 = useCallback(
+    async (erc721Address: string, from: string, to: string, tokenId: number) => {
```

Comment:
pass your `afterTransfer` param here instead of in hook init params for more flexibility in the caller right?

```suggestion
    async (erc721Address: string, from: string, to: string, tokenId: number, onSuccess: any) => {
```

Reply:
updated! 👍